lib/tpm2_eventlog_yaml.c

```diff
@@ -307,7 +307,7 @@ bool yaml_event2hdr_callback(TCG_EVENT_HEADER2 const *eventhdr, size_t size,
         return false;
     }
 
-    tpm2_tool_output("- Event[%zu]:\n", *count++);
+    tpm2_tool_output("- Event[%zu]:\n", (*count)++);
```

Comment:
This is the only non-backwards compatible change, which fixes a bug in the event numbering of `tpm2_eventlog`